addons/mail/static/src/core/web/chatter.xml

```diff
@@ -93,7 +93,12 @@
             </div>
             <t t-if="state.thread.composer.type">
                 <t t-if="state.thread.composer.type === 'message'">
-                    <div class="flex-shrink-0 px-3 pt-3 text-truncate small" style="margin-left:48px" t-out="toFollowersText"/>
+                    <div class="flex-shrink-0 px-3 pt-3 text-truncate small" style="margin-left:48px">
+                        <t t-out="toFollowersText"/>
+                        <button class="btn btn-link" t-on-click="onClickFollowerList" title="Show all recipients">
+                            <i class="fa fa-caret-down"/>
+                        </button>
```

Comment:
Not just the icon should be clickable. The inline listing too.
![Screenshot 2023-06-27 at 18 19 14](https://github.com/odoo/odoo/assets/6569390/005d028e-9cf9-4917-af0c-346cb634ee0a)

Reply:
I don't agree with this, if you have a caret, it's the caret that is clickable otherwise it's useless to have the caret, we could just display the popup over when you hover the list (witch is not a bad idea IMHO).

cc @paju-odoo

Reply:
Hi @phenix-factory @alexkuhn,

I'd prefer to keep the same behavior as Gmail, only the caret is clickable and opens the list. Having the list hover is also quite annoying when you want to type in the composer and the hover is not intentional, you'll have to come from below with your mouse. Also, most people don't need to double-check e-mail addresses and only do so when they want to. 

However, could we add a background to the caret icon on hover to make it clearer? Just a light gray, like the secondary buttons.

Reply:
In Gmail we can click on single item in inline listing, which explains why only the caret is clickable.
IMO it feels a bit better when the clickable area covers whole inline listing.
The showing of caret is more a visual cue that we can interact with is to display a dropdown item, not necessarily the precise and only UI element that is clickable.

This is a small detail, so if you don't agree we can keep click just on arrow for now.

Reply:
I just tested in Gmail, and only the caret is clickable and will display all info. 
You can hover on one single recipient, and you'll see his details, but that's another feature.